packages/cyberstorm/src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import React, { CSSProperties, ReactNode } from "react";
+import styles from "./Toast.module.css";
+import { Icon } from "../Icon/Icon";
+import { classnames } from "../../utils/utils";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faXmarkLarge } from "@fortawesome/pro-solid-svg-icons";
+import * as RadixToast from "@radix-ui/react-toast";
+
+type _ToastProps = {
+  id: string;
+  content: ReactNode;
+  icon?: JSX.Element;
```

Comment:
Looking at the designs, I would remove this prop and infer the icon internally from the `variant` prop. It would make the usage of the component easier/cleaner. And while it would limit us to predefined icons, that could also be seen as plus since it keeps the UI consistent.